bootstrap_customizer/utils.py

```diff
@@ -51,6 +52,8 @@ def generate_above_the_fold_css(theme):
     """
     context = get_scss_template_context(theme)
     scss_string = render_to_string('bootstrap_customizer/bootstrap_above_the_fold.scss', context)
+    if platform == "win32" or platform == "cygwin":
+        scss_string = scss_string.replace("\\", "/")
```

Comment:
@vincent-picot Would we be able to move the path separators to constants? It would make things a little easier to understand to anyone new to the code base. The constants could be:

```
POSIX_PATH_SEPARATOR = '/'
WINDOWS_PATH_SEPARATOR = '\\'
```